src/components/Lobby/Lobby.tsx

```diff
@@ -61,6 +61,7 @@ function Lobby({ players }: Props) {
   };
   //@ts-ignore
   const playerNames = playerList.map((player) => {
+    console.log(player)
```

Comment:
I initially changed the way the players' names were accessed after noticing that as the creator of a game, my name, nor other players' names were displayed in the `players` tab. After console logging what data was being accessed for each`player` I realized that the structure of the data changed after the `Lobby` component's state was updated by the player_list data emitted by the Action Cable.
The app if failing to establish a connection to the Action Cable locally(see below) but is successfully connecting in development, so the previously failing tests pass, but this change doesn't operate as expected in development.
<img width="684" alt="Screenshot 2024-08-22 at 12 16 17 AM" src="https://github.com/user-attachments/assets/b1a6754c-c306-4b3d-9226-a2f71885311b">

Reply:
Everything is resolved in development now - will disregard the failed local websocket connection until we can connect with BE team!